### DISCUSSION TITLE
How to change font size of Table Data in Node Red?

### DISCUSSION THREAD

**Discussion by techylax** [zachowj/hass-node-red#185] [Q&A]
Hello everyone,

I'm new to Node-RED and I'm currently facing a challenge with adjusting the font size in my table data. The HMI UI I'm working on lacks optimization, and I've made several attempts to resolve this issue without success.

I've experimented with different approaches, including utilizing the Function and Template nodes and incorporating CSS within them. However, I haven't achieved the desired outcome so far.

If anyone could offer assistance with this matter, I would greatly appreciate it. Any suggestions or guidance on how to effectively change the font size in my table data would be immensely helpful. Thank you for your time and support

![fffff](https://github.com/zachowj/hass-node-red/assets/22542035/266fa10d-b311-4920-a7a4-8632e20d2a04)
![ttttttt](https://github.com/zachowj/hass-node-red/assets/22542035/9779b7eb-556a-4957-8d2c-a6034a38682c)

**Comment by zachowj** [comment]
Try asking in the Node-RED discourse. 

https://discourse.nodered.org/

**Reply by techylax** [reply]
Thank you so much.
